_inc/client/setup-wizard/index.jsx

```diff
@@ -2,7 +2,7 @@
  * External dependencies
  */
 import React from 'react';
-import { Redirect, Route, Switch, useRouteMatch } from 'react-router-dom';
+import { Redirect, Route, Switch, useLocation, useRouteMatch } from 'react-router-dom';
```

Comment:
It looks like `useLocation` and `useRouteMatch` aren't used. Can they be removed?

```suggestion
import { Redirect, Route, Switch } from 'react-router-dom';
```